achievements/api-platform.md

```diff
@@ -1 +1,49 @@
+# API Platform Project achievements
 
+This weekend we were multiple contributors to API Platform, some of the core team were here too to help in the hackathon.
+When it comes to **API Platform** we just merge 20 PR, 10 new opened PR, 87 closed issues and 4 news. Lots of word being done from the old and the new contributors, big kudos !!
+
+Since 2 years from now, API Platform supports subresources, this features does give access to include relation in the relation, this feature is used a lot and is very interesting, but since it's has alot of complexity and limited.
```

Comment:
```suggestion
Since 2 years from now, API Platform supports subresources, this features allows to access a relation from another, this feature is used a lot and is very interesting, but since it's has a lot of complexity and is limited.
```

Reply:
Please fix "since it's has a lot of complexity and is limited"

Reply:
also "this feature" (singular)
then "it is used a lot" (avoid repeating same word)